src/components/TeamMemberComponent.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Grid, Typography, Dialog, Card, CardActionArea, CardMedia, CardActions
+} from '@material-ui/core';
+import data from "../assets/TeamMembersData.json";
+import styles from "./TeamMembersLayout";
+const useStyles=makeStyles((theme)=>(styles(theme)));
+const team = data.members;
+
+export default function TeamMemberComponent() {
+  const classes = useStyles();
+  const [open, setOpen] = useState(false);
+  const [index, setIndex] = useState(0);
+  const unknown = "Unknown";
+  const [phone, setPhone] = useState(unknown);
+  const [email, setEmail] = useState(unknown);
+  const [github, setGithub] = useState(unknown);
+  const [linkedin, setLinkedin] = useState(unknown);
+  const handleOpen = (data) => {
+    setOpen(true);
+    if (data != undefined) {
+      setIndex(data);
+      if (team[data].phonenNo !== undefined)
+        setPhone(team[data].phonenNo);
+      if (team[data].emailid !== undefined)
+        setEmail(team[data].emailid);
+      if (team[data].githublink !== undefined)
+        setGithub(team[data].githublink);
+      if (team[data].linkedin !== undefined)
+        setLinkedin(team[data].linkedin);
+    }
```

Comment:
null checking can be done in this way:
```
if( nullVariable )  // returns true if variable is not null, else false rather than : 
if( nullVariable != undefined )
```